channels/sy.m3u

```diff
@@ -27,3 +27,5 @@ https://svs.itworkscdn.net/syriatvlive/syriatv.smil/playlist.m3u8
 http://82.137.248.16:1935/SEdu/stream10112020/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/6KUtkKz.png" group-title="News",Syrian News Channel
 http://82.137.248.16:1935/Snews/stream10112020/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/4x2uwpx.png" group-title="",Ugarit TV [not 24/7]
+http://82.137.248.16:1935/Ugarit/stream10112020/playlist.m3u8
```

Comment:
What did you use to test this, I can't get it to work - is it a part time channel?

Reply:
Just tested now, it answers. I think it's not a 24/7 channel.

Reply:
It works for half the day (whilst the channel is broadcasting).

Thanks @LaneSh4d0w